src/dsp_tools/commands/validate_data/reformat_validaton_result.py

```diff
@@ -352,25 +369,28 @@ def _reformat_one_validation_result(validation_result: ValidationResult) -> Inpu
             return _reformat_link_target_violation_result(validation_result)
         case ResultUniqueValueViolation():
             return _reformat_unique_value_violation_result(validation_result)
+        case ResultFileValueViolation():
+            iris = _reformat_main_iris(validation_result)
+            return FileValueProblem(
+                res_id=iris.res_id,
+                res_type=iris.res_type,
+                prop_name="bitstream / iiif-uri",
+                expected=validation_result.results_message,
+            )
         case _:
             raise BaseError(f"An unknown violation result was found: {validation_result.__class__.__name__}")
 
 
-def _reformat_min_cardinality_validation_result(validation_result: ResultMinCardinalityViolation) -> InputProblem:
-    iris = _reformat_main_iris(validation_result)
-    file_value_properties = ["hasMovingImageFileValue"]
-    if iris.prop_name in file_value_properties:
-        return FileValueProblem(
-            res_id=iris.res_id,
-            res_type=iris.res_type,
-            prop_name="bitstream / iiif-uri",
-            expected=validation_result.results_message,
-        )
-    return MinCardinalityProblem(
-        res_id=iris.res_id,
-        res_type=iris.res_type,
-        prop_name=iris.prop_name,
-        expected_cardinality=validation_result.results_message,
+def _reformat_with_prop_and_message(
```

Comment:
I didn't make any of the others generic, because there are too many differences.